src/asana/helpers.py

```diff
@@ -394,22 +398,26 @@ def _task_description_from_pull_request(pull_request: PullRequest) -> str:
         + "\n✍️ "
         + author
         + _generate_assignee_description(pull_request.assignee())
+        + f"\n❗️Task is {status} because {status_reason.reason}"
```

Comment:
Do you think we should include the ❗ when "the pull request is open"? It seems like an alert that something is wrong, when it might not be... (I realize this was probably the original logic in the clojure code, but curious what your thoughts are... it does make the code a bit cleaner, so I think it's not the end of the world)

Reply:
I can see keeping the ❗️ as a visual indication of where to look if I want to see the status of this PR. Having it go away might cause some brief confusion regarding where to find the PR status in the body. I'm okay with either option though!

Reply:
We can start with it and see if we get feedback, can always change it later